.github/workflows/call-r-cmd-check.yml

```diff
@@ -9,4 +9,8 @@ on:
     #- cron: '0 0 * * 0'
 jobs:
   call-workflow:
+    timeout-minutes: 60
     uses: nmfs-fish-tools/ghactions4r/.github/workflows/r-cmd-check.yml@main
+    with: 
+      gha_timeout_minutes: 30
```

Comment:
This is just a question, not change request. Does this section of code mean that the overall timeout minutes for the call-r-cmd-check workflow is 60 mins and the timeout minutes for r-cmd-check workflow is 30 mins?

Reply:
Nope, this is just a mistake! I should just be using the part after with: and intended to put it to 60 min. Good catch!